pkg/services/ngalert/remote/forked_alertmanager_test.go

```diff
@@ -376,25 +376,41 @@ func TestForkedAlertmanager_ModeRemotePrimary(t *testing.T) {
 	expErr := errors.New("test error")
 
 	t.Run("ApplyConfig", func(tt *testing.T) {
-		{
```

Comment:
I just removed this brace and the closing one, the test for this method is otherwise unchanged